util/src/Pos/Util/Log/Internal.hs

```diff
@@ -42,6 +50,20 @@ s2kname s = K.Namespace [s]
 s2knames :: [Text] -> K.Namespace
 s2knames s = K.Namespace s
 
+-- | log files have a prefix and a name
+data FileDescription = FileDescription {
+                         prefixpath :: FilePath,
+                         filename   :: FilePath }
+                       deriving (Show)
+
+mkFileDescription :: FilePath -> FilePath -> FileDescription
+mkFileDescription bp fp =
+    -- if fp contains a filename in a directory path
+    --    move this path to the prefix and only keep the name
+    let (extbp, fname) = splitFileName fp
```

Comment:
some log-config files contain as filename a path like this: `pub/node.pub`
in that case, we add the directory path to the prefix dir and keep the filename apart.